mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -599,6 +600,26 @@ public void showUploadImageProgressbar(boolean b) {
         }
     }
 
+
+    public void loadImageGlide(int clientId) {
+        pb_imageProgressBar.setVisibility(VISIBLE);
+        String url = PrefManager.getInstanceUrl()
+                + "clients/"
+                + clientId
+                + "/images?maxHeight=120&maxWidth=120";
+
+        GlideUrl glideUrl = new GlideUrl(url, new LazyHeaders.Builder()
```

Comment:
Similarly, you can also take this glideURL thing out of it and put it in the same class that gives you the String URL. So basically, you would never want to expose the URL at first and what should be exposed via a public method is the glideURL to the view that wants to show the Image.

Please let me know if you didn't understand any part of the review, I'd be happy to help.